src/main/resources/bin/windup

```diff
@@ -57,7 +57,7 @@ while [ "$1" != "" ] ; do
     WINDUP_DEBUG_ARGS="-Xdebug -Xnoagent -Djava.compiler=NONE -Xrunjdwp:transport=dt_socket,server=y,suspend=y,address=8000"
   fi
 
-  QUOTED_ARGS="$QUOTED_ARGS \"$1\""
+  QUOTED_ARGS="$QUOTED_ARGS $1"
```

Comment:
The quotes are necessary for "eval" because you have to arrange the arguments as if they were being manually input on the command line.

Having said that, I'm really not sure how this new format deals with spaces on these parameters. I'm a little worried that the approach isn't quite right here.